index.html

```diff
@@ -1,3 +1,50 @@
-﻿
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta name="description" content="Demo project using HTML, CSS, & JavaScript">
 
+    <title>GranniezCookiez</title>
+  </head>
 
+  <body>
+    <header>
+      <h1>The Best Chocolate Chip Cookies</h1>
+    </header>﻿
+<img src="./img/cookies.jpg" alt="grandmascookies">
+
+    <section>
+      <h2>Recipe By Grandma</h2>
+    <section>
+      <h2>Prep Time: 45 MIN</h2>
+    </section>
+    <section>
+      <h2>--</h2>
+    </section>
+      <h2>INGREDIENTS:</h2>
+    <ul>
+      <li>1 1/2 cup (3 sticks) softened butter</li>
+      <li>1 cup brown sugar</li>
+      <li>1 cup granulated sugar</li>
+      <li>1 Tbl vanilla instant pudding powder</li>
+      <li>2 Tbl milk</li>
+      <li>2 Tbl vanilla extract</li>
+      <li>2 eggs</li>
+      <li>4 cups chocolate chips</li>
+      <li>1 cup chopped walnuts or pecans (optional)</li>
+    </ul>
+      <h2>How to make Mama's Recipe:</h2>
+    <ol>
+      <li>Preheat oven to 350 degrees</li>
+      <li>Beat butter and sugars together until light and fluffy</li>
+      <li>Stir in pudding mix milk and vanilla extract</li>
+      <li>Beat in eggs</li>
+      <li>Add dry ingredients and stir until combined</li>
+      <li>Stir in chocolate chips and nuts</li>
+      <li>Place 1 1/2 inch balls of dough 2 inches apart on an ungreased cookie sheet</li>
+      <li>Bake 8-10 minutes or until golden brown</li>
+      </ol>
+      <h2>Nutrition Information:</h2>
+      <p>Probably bad for you!</p>
+      <h2>Inspiration Recipe:</h2>
+      <a href="http://www.opensourcefood.com/people/Amanori/recipes/mamas-recipe-the-best-chocolate-chip-cookies">Inspiration Recipe</a>
+      <p>--</p>
```

Comment:
the browser is forgiving, but don't forget to close the `body` and `html` tags with `</body>` and `</html>`